src/index.js

```diff
@@ -14,19 +14,19 @@ function App() {
     setCompanyName(value);
   }
 
-  const [nameCeo, setNameCeo] = React.useState(localStorage.getItem("nameCeo"));
+  const [ceoName, setCeoName] = React.useState(localStorage.getItem("ceoName"));
 
-  function handleNameCeo(value) {
-    localStorage.setItem("nameCeo", value);
-    setNameCeo(value);
+  function handleCeoName(value) {
+    localStorage.setItem("ceoName", value);
```

Comment:
This code repeats each time we save to `localStorage`, we have to "streamline" the app later though.